index.html

```diff
@@ -27,11 +27,37 @@ <h1 id="" class="">Las paisas</h1>
 		</article>
 	</section>
 	<section>
-		<!-- comodities section -->
+    <!-- comodities section -->
+    <div id="carrouselContainer" class="carrousel-container">
+      <h1 class="title-gallery">Galeria</h1>
+      <hr class="line">
+      <div class="description">
+        <p>
+          Nuestro hostal te ofrece todas las comodidades para que tengas la
+          mejor experiencia, observa todo lo que tenemos para ti y lo
+          majestuoso de Capurganá 
+        </p>
```

Comment:
why are the `title, line` and `p` inside the carrousel container if they are not carrousel items?